staging/src/k8s.io/endpointslice/utils_test.go

```diff
@@ -585,6 +586,51 @@ func TestGetEndpointPorts(t *testing.T) {
 				AppProtocol: pointer.String("https"),
 			}},
 		},
+		"service with named port for restartable init containers": {
```

Comment:
can you add one test case for the sidecar and the normal container to use the same named port, and to validate the normal container takes precedence

Reply:
Thanks for your review! I added a test case at 4e21f535475e27fbb0b558a00de6d0d743ac3906, does this align with what you meant?